environment.yml

```diff
@@ -5,3 +5,4 @@ channels:
 dependencies:
   - python=3
   - folium
+  - nb_black=1.0.5=py_0
```

Comment:
@MarcoGorelli do we need to pin the version?

Reply:
I guess not

Reply:
(nobody expects the version number!)